src/request/components/RequestDetailPanelData.tsx

```diff
@@ -0,0 +1,48 @@
+import { MasterDetailTriPanel } from './MasterDetailTriPanel';
+import { RequestDetailPanelDataOperationCommandContainer } from '../containers/RequestDetailPanelDataOperationCommandContainer';
+import { RequestDetailPanelDataOperationTableContainer } from '../containers/RequestDetailPanelDataOperationTableContainer';
+
+import * as React from 'react';
+import Highlight = require('react-highlight');
+
+export interface IDataComponentProps {
+    totalOperationCount: number;
+}
+
+export class DataComponent extends React.Component<IDataComponentProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data'>
+            {
+                this.props.totalOperationCount > 0
+                    ? <MasterDetailTriPanel masterPanel={this.renderMaster()} leftDetailPanel={this.renderLeftDetail()} leftDetailPanelTitle='Command' rightDetailPanel={this.renderRightDetail()} rightDetailPanelTitle='Summary' />
+                    : <div className='tab-section text-minor'>Could not find any data.</div>  
+            }
+            </div>
+        );
+    }
```

Comment:
Not for now, but as a more general thing, I wonder if the "empty" case should be handled by the target component? I would mean that all the sites that use it don't have to have this repeated logic.